src/sqlancer/questdb/QuestDBProvider.java

```diff
@@ -59,6 +61,10 @@ private static int mapActions(QuestDBGlobalState globalState, Action a) {
             return r.getInteger(0, 3);
         case TRUNCATE:
             return r.getInteger(0, 5);
+        case UPDATE:
```

Comment:
I think we should only add new features once we have a CI/CD that works for a given QuestDB version.